modules/mobile-agents/android/jax-rs/src/main/webapp/WEB-INF/cxf-servlet.xml

```diff
@@ -32,7 +32,7 @@
             <ref bean="errorHandler"/>
         </jaxrs:providers>
     </jaxrs:server>
-    <jaxrs:server id="deviceManagementService" address="/devices">
+    <jaxrs:server id="deviceManagementService" address="/device">
```

Comment:
The reason that I've changed here is to maintain the consistency across all the JAX-RS implementation.